code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -32,6 +32,7 @@
 	w_class = W_CLASS_LARGE
 	storage_slots = 14
 	max_combined_w_class = 200 //This actually doesn't matter as long as it is arbitrarily high, bar will be set by storage slots
+	fits_max_w_class = 3
```

Comment:
You should use w_class defines for this if you're going to make this kind of change.